README.md

```diff
@@ -5,22 +5,55 @@ Add DTrace probes to Diesel connections.
 ## Overview
 
 The `diesel-dtrace` crate provides a diesel [`Connection`][1] that includes DTrace probe points.
-Probes are fired when a connection to the database is established and for each query.
+Probes are fired when a connection to the database is established and for each
+query and transaction.
```

Comment:
1. this is fine
2. do you want to be more explicit? i.e. Probes fire before and after connections, quries, and transactions

Reply:
Done in 0358fa4